config/locales/fr.yml

```diff
@@ -516,9 +516,9 @@ fr:
     already_ordered:
       cart: "panier"
       message_html: "Vous avez déjà passé une commande pour ce cycle de vente. Vérifiez votre %{cart} pour voir les produits commandés. Vous pouvez annuler ou modifier votre commande jusqu'à la fermeture du cycle de vente."
-  home:
+  shops:
```

Comment:
We must add this then at the top-level documentation of the `en.yml` that @oeoeaio added. It gives some guidelines when dealing with translations in OFN.